File: mantente-app/fix_ventas.js
const fs = require('fs');
const path = require('path');

const filePath = 'src/components/Ventas.jsx';

try {
  let content = fs.readFileSync(filePath, 'utf8');
  content = content.replace(/crearFactura/g, 'createFactura');
  content = content.replace(/crearCliente/g, 'createCliente');
  fs.writeFileSync(filePath, content, 'utf8');
  console.log('✅ Ventas.jsx actualizado');
} catch (err) {
  console.error('Error:', err.message);
}

File: mantente-app/fix_ventas.py
#!/usr/bin/env python3
import re

file_path = r'c:\Users\angel\OneDrive\Documents\proyecto mantente\mantente-app\src\components\Ventas.jsx'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('crearFactura', 'createFactura')
content = content.replace('crearCliente', 'createCliente')

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print("✅ Ventas.jsx actualizado")
